### PR TITLE
Allow clinical to run >1 script

### PR DESCRIPTION
**Pull request by wajiha-oicr**
**Description of changes**

- Instead of running one script, it loops through an array of scripts, breaking on the first one that returns an invalid result. 
- If somehow a string is still passed, it's placed inside an array first (backwards-compatibility) 
___
- I think the tsc linter reformatted some things too
- I think Jenkins is failing because of [npm being down ](https://status.npmjs.org/)

**Type of Change**

- [ ] Bug
- [ ] Refactor
- [x] New Feature

**Checklist before requesting review:**

- [x] Check branch (code change PRs go to `develop` not master)
- [x] Manual testing
- [ ] Regression tests completed and passing (double check number of tests).
- [ ] Spelling has been checked.
- [x] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

**Comment by joneubank** [comment]
Non-issue for this PR, but in the future could you use descriptive branch names?

File: src/lectern-client/schema-functions.ts [hunk 1]
```diff
@@ -505,22 +505,44 @@ namespace validation {
...
+        $name: field.name,
       };
 
       if (!field.restrictions || !field.restrictions.script) {
         throw new Error('called validation by script without script provided');
       }
 
-      const script = new vm.Script(field.restrictions.script);
+      // scripts should already be strings inside arrays, but ensure that they are to help transition between lectern versions
+      // checking for this can be removed in future versions of lectern (feb 2020)
```
Comment (joneubank) [inline src/lectern-client/schema-functions.ts]: future versions of clinical 😆

File: src/lectern-client/schema-functions.ts [hunk 2]
```diff
@@ -505,22 +505,44 @@ namespace validation {
...
+      const scripts =
+        typeof field.restrictions.script === 'string'
+          ? [field.restrictions.script]
+          : field.restrictions.script;
```
Comment (joneubank) [inline src/lectern-client/schema-functions.ts]: 👍

**Comment by blabadi** [comment]
@joneubank @wajiha-oicr  This PR was merged with failing tests

**Comment by wajiha-oicr** [comment]
> @joneubank @wajiha-oicr This PR was merged with failing tests

should I revert, and wait until the npm registry issue is fixed? (ran npm run test on my local branch, all tests pass)

**Pull request merged**